udata/static/js/widgets/share-btn.js

```diff
@@ -16,11 +17,17 @@ define([
             title: i18n._('Share'),
             placement: 'top',
             container: 'body',
-            trigger: 'focus',
+            // trigger: 'focus', // Doesn't work on OSX+FF/iOS+Safari
             content: tpl({
                 url: $this.data('share-url'),
                 title: $this.data('share-title')
             })
+        }).on('shown.bs.popover', function() {
+            $(document.body).find('.share-click').each(function() {
```

Comment:
Unfortunately, I can't because `this` refers to the clicked element (`btn-share`) and links are appended at the end of the DOM, just before the `</body>`